android/src/main/java/com/fidelreactlibrary/adapters/FidelSetupKeys.java

```diff
@@ -53,6 +55,25 @@ public String jsName() {
         }
     }
 
+    public enum CardConfig {
+        ID("id"),
```

Comment:
These fields can actually be removed from the FidelSetupKeys enum, for 2 reasons:
1. This enum is used for the setup function (hence it's name includes `setup`, as it contains the keys used in the Fidel.setup function). The card verification configuration property keys are not used in the Fidel.setup function.
2. This object is already big. What if we introduce a new function, do we keep adding properties here?

I would suggest creating a new:
```
public enum CardVerificationConfigurationKeys {
ID("id"),
CONSENT_ID("consentID"),
LAST_4_DIGITS("last4Digits");
}
```
And in the adapter you can use this one, instead of the FidelSetupKeys enum:
CardVerificationConfigurationKeys.ID